Agreements/Internal/Wal_Nor.txt

```diff
@@ -0,0 +1,5 @@
+COPX:*:*:POL:EGNM:*:Wallasey:North:*:15000:BARTN5
+COPX:*:*:BARTN:EGCN:*:Wallasey:North:*:15000:BARTN5
+
+COPX:EGGP:*:POL:*:*:Wallasey:North:*:9000:^POL
```

Comment:
Is this one of the ones where we might want to use Adam's clever 'fix after'? POL is quite a way into North's sector, so it might be sensible to stop the agreement from displaying beyond transfer of the tag.